File-Formats/DocIO/Working-with-Word-document.md

```diff
@@ -1432,6 +1432,209 @@ using (WordDocument document = new WordDocument(assembly.GetManifestResourceStre
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-document/Apply-picture-background-to-document).
 
+## Hide background in print layout view
+
+You can show or hide background colors and images in print layout view of Word document using `DisplayBackgrounds` API.
+
+The following code example shows how to hide background in print layout view of Word document.
```

Comment:
The following code example shows how to hide the background in the print layout view of a Word document:

Reply:
Addressed